test/powershell/Host/TabCompletion/BugFix.Tests.ps1

```diff
@@ -35,4 +35,18 @@ Describe "Tab completion bug fix" -Tags "CI" {
         $result.CompletionMatches[1].CompletionText | Should Be "-NoClobber"
         $result.CompletionMatches[2].CompletionText | Should Be "-NoOverwrite"
     }
+    It "Issue#3416 - 'Select-Object -ExcludeProperty <tab>' should work" {
```

Comment:
This file is the only one where we have issues numbers in the output of the test - I'm not sure how useful that is though.

I find `git blame` sufficient for tracking back to an issue, but comments are also sufficient. In the output though - it's just clutter in my opinion.